guardrails/run/runner.py

```diff
@@ -521,14 +517,8 @@ def call(
             llm_response = LLMResponse(output=output)
         elif api_fn is None:
             raise ValueError("API or output must be provided.")
-        elif msg_history:
```

Comment:
this probably needs to come back because were not actually dropping support this release